pkg/controllers/bigtableautoscaler_controller.go

```diff
@@ -102,18 +98,23 @@ func (r *BigtableAutoscalerReconciler) Reconcile(req ctrl.Request) (ctrl.Result,
 		r.fetcherStarted = true
 	}
 
-	clusters, err := getClusters(credentialsJSON, "cdp-development", "clustering-engine")
+	googleCloudClient, err := googlecloud.NewClient(ctx, credentialsJSON, "cdp-development", "clustering-engine", "clustering-engine-c1")
+
+	if err != nil {
+		return ctrl.Result{}, err
+	}
+
+	currentNodes, err := googleCloudClient.GetCurrentNodeCount()
```

Comment:
It doesn't make sense for our `googlecloud` client to have a cluster ID. WDYT about passing the cluster ID as a parameter to `GetCurrentNodeCount()` and `GetLastCPUMeasure()` instead of having it at the prototype level?

Reply:
Could you explain why it doesn't make sense?
 This information is part of the spec and defines each individual bigtable resource being managed.

Reply:
Sorry, I should've elaborated better. Every Google Cloud component has a project ID and an instance ID, but cluster ID is specific to the bigtable scope. Since it is a Google Cloud client, I think its better if it only needs google cloud parameters to be initialized. If we someday add a method to check a quota, for example, it won't need the cluster ID.